python/ambassador/envoy/v3/v3listener.py

```diff
@@ -906,15 +906,15 @@ def compute_http_routes(self) -> None:
                             extra_info = " (force Route for ACME challenge)"
                             action = "Route"
                             found_acme = True
-                        elif (
-                            self.config.ir.edge_stack_allowed
-                            and (route_precedence == -1000000)
-                            and (
-                                rv.route["match"].get("safe_regex", {}).get("regex", None) == "^/$"
-                            )
-                        ):
-                            extra_info = " (force Route for fallback Mapping)"
-                            action = "Route"
+                        # elif (
```

Comment:
Leaving this here to come back to?

Reply:
More accurately leaving it to take a closer look at to see if there's anything there that would be smart to keep. Next chainsaw PR for that stuff. 🙂